.github/workflows/sdk-update-test.yml

```diff
@@ -0,0 +1,102 @@
+name: test-docs-examples
+
+on:
+  repository_dispatch:
+    types: [sdk-update]
+
+jobs:
+  workshop:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+      - uses: datamonsters/replace-action@v2
+        with:
+         files: 'code-examples/workshop/package.json'
+         replacements: '${{ github.event.client_payload.previousTag }}=${{ github.event.client_payload.latestTag }}'
+      - name: Test style conventions
+        working-directory: code-examples/workshop
+        run: |
+          yarn install --frozen
```

Comment:
did you do a test run? I'm asking because I would expect this to fail with the `--frozen` option after having changed the package.json

Reply:
removed